Classes.py

```diff
@@ -57,30 +160,125 @@ def __init__(self, tupPhases):
             tupPhases: list of phase classes
         
         """
+        
+        pctResidual = .01 # value from slides
+        
         # Initialize variables 
         mPropImpulse     = 0
         mPropImpulseOx   = 0
         mPropImpulseFuel = 0
 
+        mPropImpulseMono = 0
+        mPropImpulseReserve = 0
+        mPropImpulseReserveOx =0
+        mPropImpulseReserveFuel = 0
+        mPropImpulseReserveMono = 0
+        mPropBoiloff     = 0
+        mPropBoiloffOx   = 0
+        mPropBoiloffFuel = 0
+        mPropRCS         = 0
+        
+        mPropChill     = 0
+        mPropChillOx   = 0
+        mPropChillFuel = 0
+        
+        mPropSettling = 0
+        
         # sum up the usages by phase
         for curPhase in tupPhases:
             mPropImpulse     += curPhase.mPropImpulse 
             mPropImpulseOx   += curPhase.mPropImpulseOx
             mPropImpulseFuel += curPhase.mPropImpulseFuel
 
+            mPropImpulseMono += curPhase.mPropImpulseMono
+            
+            mPropImpulseReserve +=  curPhase.mPropImpulseReserve
+            mPropImpulseReserveOx +=  curPhase.mPropImpulseReserveOx
+            mPropImpulseReserveFuel += curPhase.mPropImpulseReserveFuel
+            mPropImpulseReserveMono +=  mPropImpulseReserveMono
+            
+            mPropBoiloff     +=  curPhase.mPropBoiloff
+            mPropBoiloffOx   +=  curPhase.mPropBoiloffOx
+            mPropBoiloffFuel +=  curPhase.mPropBoiloffFuel
+            mPropRCS         +=   curPhase.mPropRCS
+            
+            mPropChill       += curPhase.mPropChill
+            mPropChillOx     += curPhase.mPropChillOx
+            mPropChillFuel   += curPhase.mPropChillFuel
+            
+            mPropSettling    += MissionSummary.mPropSettling
+
+        # Stuff everything into self    
+        mPropConsumedOx   = mPropImpulseOx   + mPropBoiloffOx + mPropChillOx
+        mPropConsumedFuel = mPropImpulseFuel + mPropBoiloffFuel + mPropChillFuel
+        mPropConsumedMono = mPropImpulseMono + mPropRCS + mPropSettling
+        mPropConsumedTotal = mPropConsumedOx + mPropConsumedFuel + mPropConsumedMono
+        
+        mPropResidualOx    = MissionSummary.pctResdiual*(mPropConsumedOx+mPropImpulseReserveOx)
```

Comment:
This should be just pctResdiual instead of MissionSummary.pctResdiual (same for the next few lines)